.github/workflows/fuzz.yml

```diff
@@ -15,6 +15,17 @@ on:
   schedule:                                   # Production runs against default branch.
     - cron: '24 11 * * 0'                     # Run after 11:24 UTC (4:24am PDT/3:24am PST) on Sun.
   workflow_dispatch:                          # Manual runs.
```

Comment:
Odd to see this syntax with `workflow_dispatch:` having no children/items and `push:` being a sibling immediately beneath it. If it's valid yaml and works, looks good to me!